envoystat/envoystat.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+import argparse
+import datetime
+import sys
+import signal
+import time
+import urllib2
+import urlparse
+
+
+GAUGE_NOTATION = "-"
+TIMEOUT_SECONDS = 0.1
+
+SHOULD_QUIT = False
+
+
+def signal_handler(signal, frame):
+    global SHOULD_QUIT
```

Comment:
I think all uppercase is only for "constants" (by convention) rather than mutable vars, so maybe just `should_quit`?